arbitrum-docs/how-arbitrum-works/timeboost/gentle-introduction.mdx

```diff
@@ -0,0 +1,142 @@
+
+- Have the right to re-order transactions.
+- Have a guarantee that their transactions will always be first at the “top-of-the-block.”
+- Guarantee a profit at all.
+
+The value of the express lane will be the sum of how much MEV the express lane controller predicts they can extract during the upcoming round (i.e., MEV opportunity estimates made before the auction closes) _plus_ the amount of MEV extracted by the express lane controller while they are in control (that they otherwise did not predict). Understanding how the value of the express lane is determined can be useful for chain owners when adjusting to the artificial delay and the time before the auction closes.
+
+### The Timeboost auction
+
+Control of the express lane in each round (default: 60 seconds) is determined by a per-round auction, which is a sealed-bid, second-price auction. This auction is held to determine the express lane controller for the next round. In other words, the express lane controller was determined at any point in time in the previous auction round. Bids for the auction can be made with any ERC20 token, in any amount, and be collected by any address - at the full discretion of the chain owner.
+
+![timeboost auction workflow](./images/centralized-timeboost-auction-workflow.jpg)
+
+The auction for a round has a closing time that is `auctionClosingSeconds` (default: 15) seconds before the beginning of the round. This means that, in the default parameters, parties have 45 seconds to submit bids before the auction will no longer accept bids. In the 15 seconds between when bids are no longer accepted and when the new round begins, the autonomous auctioneer will verify all bids, determine the winner, and make a call to the on-chain auction contract to formally resolve the auction.
+
+### Auction contract
+
+Before bidding in the auction, a party must deposit funds into the Auction Contract. Deposits can be made, or funds can be added to an existing deposit at any time. There is no minimum deposit amount, but there is a starting minimum bid of 0.001 ETH (default amount and token). These deposits are fully withdrawable, with some nominal delay (2 rounds, or 2 minutes by default), so as not to impact the outcome of an existing round.
```

Comment:
How likely is the default minimum bid to change?
If the plan is to update it depending on usage, I would maybe change the language of this sentence
`but there is a starting minimum bid of 0.001 ETH (default amount and token)`
to something like
`but there is a defined minimum bid, which by default starts at 0.001 ETH`